composer.json

```diff
@@ -35,17 +35,17 @@
         "doctrine/orm": "^2.5",
         "sensio/distribution-bundle": "^5.0.22",
         "incenteev/composer-parameter-handler": "^2.1.3",
-        "ezsystems/ezpublish-kernel": "^7.2@dev",
-        "ezsystems/repository-forms": "^2.2@dev",
-        "ezsystems/ezplatform-solr-search-engine": "^1.5@dev",
-        "ezsystems/ez-support-tools": "^0.2@dev",
-        "ezsystems/ezplatform-http-cache": "^0.7@dev",
-        "ezsystems/ezplatform-admin-ui": "^1.2@dev",
-        "ezsystems/ezplatform-admin-ui-modules": "^1.2@dev",
-        "ezsystems/ezplatform-admin-ui-assets": "^3.0@dev",
-        "ezsystems/ezplatform-design-engine": "^2.0@dev",
-        "ezsystems/ezplatform-standard-design": "^0.1@dev",
-        "ezsystems/ezplatform-cron": "^2.0@dev",
+        "ezsystems/ezpublish-kernel": "~7.2.0@dev",
+        "ezsystems/repository-forms": "~2.2.0@dev",
+        "ezsystems/ezplatform-solr-search-engine": "~1.5.0@dev",
+        "ezsystems/ez-support-tools": "~0.2.1@dev",
+        "ezsystems/ezplatform-http-cache": "~0.7.0@dev",
+        "ezsystems/ezplatform-admin-ui": "~1.2.0@dev",
+        "ezsystems/ezplatform-admin-ui-modules": "~1.2.0@dev",
+        "ezsystems/ezplatform-admin-ui-assets": "~3.0.0@dev",
+        "ezsystems/ezplatform-design-engine": "~2.0.0@dev",
+        "ezsystems/ezplatform-standard-design": "~0.1.0@dev",
+        "ezsystems/ezplatform-cron": "~2.0.0@dev",
```

Comment:
I think this changes should come when ezplatform is merged to ezplatform-demo (should not be set directly here).